Core/GDCore/Extensions/Metadata/ParameterMetadata.h

```diff
@@ -29,6 +29,12 @@ class GD_CORE_API ParameterMetadata {
   ParameterMetadata();
   virtual ~ParameterMetadata(){};
 
+  /**
+   * \brief Return a pointer to a new ParameterMetadata constructed from
+   * this one.
+   */
+  ParameterMetadata* Clone() const { return new ParameterMetadata(*this); };
```

Comment:
Usually, Clone is only useful for class that can be derived and need a "polymorphic clone".

Reply:
If I remove it, it makes this error:
```
GDCore/Tools/SerializableWithNameList.inl:191:18: note: in instantiation of function template specialization 'gd::Clone<gd::ParameterMetadata>' requested here
  elements = gd::Clone(other.elements);
```
because of this:
```C++
class GD_CORE_API ParameterMetadataContainer
    : private SerializableWithNameList<gd::ParameterMetadata>
```

Reply:
Ok fair 👍